jekyll/_cci2/jvm-heap-size-configuration.md

```diff
@@ -0,0 +1,40 @@
+# Configurable Java virtual Machine Heap Size
+
+JVM_HEAP_SIZE is configurable for frontend and test-result containers
+
+## Setting up
+To customize JVM_HEAP_SIZE value, you will need to create customizations file in your services box
+1. Create customizations files for frontend or test-result
+```sh
+/etc/circleconfig/frontend/customizations
+/etc/circleconfig/test-results/customizations
+```
+
+2. In the file, add the line below to export desire JVM_HEAP_SIZE(e.g. 2g, 512m) in the file
+```sh
+export JVM_HEAP_SIZE=<desiredsize>
```

Comment:
I always like to include an example when possible.  In this case maybe something like:
`export JVM_HEAP_SIZE=2g`